jenkins/Jenkinsfile-publish

```diff
@@ -0,0 +1,48 @@
+// used by https://warpdrive-lab.dev.symphony.com/jenkins/view/DevX-Basilic/job/bdk-deploy-java/
+
+@Library(['sym-pipeline']) _
+
+import com.symphony.cicd.SymphonyCICDUtils
+
+cicdUtils = new SymphonyCICDUtils()
+
+
+node {
+
+    // default org and branch
+    def org = env.PARAM_REPO_OWNER ?: 'SymphonyPlatformSolutions'
+    def branch = env.PARAM_BUILD_BRANCH ?: 'master'
+
+    withCredentials([
+        file(credentialsId: 'JAVA_SDK_GPG_KEY', variable: 'ORG_GRADLE_PROJECT_KEY'),
+        string(credentialsId: 'JAVA_SDK_GPG_PASSPHRASE', variable: 'ORG_GRADLE_PROJECT_PASSPHRASE')
+    ]) {
+    
+      withEnv([
+        "PROJECT_TYPE=java",
+        "USE_OPENJDK11=false",
```

Comment:
given we are not using much of pipeline lib we could get rid of JDK, SONAR variables